assets/js/sections/qso.js

```diff
@@ -1019,18 +1024,34 @@ $("#callsign").on("keypress", function(e) {
 // On Key up check and suggest callsigns
 $("#callsign").keyup(function() {
 	if ($(this).val().length >= 3) {
-	  $('.callsign-suggest').show();
-	  $callsign = $(this).val().replace('Ø', '0');
-	  $.ajax({
-		url: 'lookup/scp',
-		method: 'POST',
-		data: {
-		  callsign: $callsign.toUpperCase()
-		},
-		success: function(result) {
-		  $('.callsign-suggestions').text(result);
-		}
-	  });
+	  let call = $(this).val().toUpperCase();
+
+      if ( scp_data.request == "" || ! call.startsWith(scp_data.request) )  {
```

Comment:
`===`

Reply:
See the other comment: As `scp_data.request` always is a string, `==` is okay here. In addition, if it is not a string, then `startsWith()` coerces it to string, which will give the correct behaviour.

Reply:
it should be `===` always use `===`